allensdk/brain_observatory/behavior/behavior_ophys_session.py

```diff
@@ -10,6 +10,52 @@
 
 
 class BehaviorOphysSession(LazyPropertyMixin):
+    """Represents data from a single Visual Behavior Ophys imaging session.  LazyProperty attributes access the data only on the first demand, and then memoize the result for reuse.
+    
+    Attributes:
+        ophys_experiment_id : int (LazyProperty)
+            Unique identifier for this experimental session
+        segmentation_mask_image : SimpleITK.Image (LazyProperty)
+            2D image of the segmented regions-of-interest in the field of view
+        stimulus_timestamps : numpy.ndarray (LazyProperty)
+            Timestamps associated the stimulus presentations on the monitor 
+        ophys_timestamps : numpy.ndarray (LazyProperty)
+            Timestamps associated with frames captured by the microscope
+        metadata : dict (LazyProperty)
+            A dictionary of session-specific metadata
+        dff_traces : pandas.DataFrame (LazyProperty)
+            The traces of dff organized into a dataframe; index is the cell roi ids
+        cell_specimen_table : pandas.DataFrame (LazyProperty)
+            Cell roi information organized into a dataframe; index is the cell roi ids
+        running_speed : allensdk.brain_observatory.running_speed.RunningSpeed (LazyProperty)
+            NamedTuple with two fields
+                timestamps : numpy.ndarray
+                    Timestamps of running speed data samples
+                values : np.ndarray
+                    Running speed of the experimental subject (in cm / s).
+        running_data_df : pandas.DataFrame (LazyProperty)
+            Dataframe containing various signals used to compute running speed
+        stimulus_presentations : pandas.DataFrame (LazyProperty)
+            Table whose rows are stimulus presentations and whose columns are presentation characteristics. 
+                    A stimulus presentation is the smallest unit of distinct stimulus presentation and lasts for 
+                    (usually) 1 60hz frame.
+        stimulus_templates : dict (LazyProperty)
+            A dictionary containing the stimulus images presented during the session keys are data set names, and values are 3D numpy arrays.
+        licks : pandas.DataFrame (LazyProperty)
+            A dataframe containing lick timestamps
+        rewards : pandas.DataFrame (LazyProperty)
+            A dataframe containing lick timestamps
+        task_parameters : dict (LazyProperty)
+            A dictionary containing task metadata
```

Comment:
Use of the word metadata here is a bit confusing (since there is also a metadata attribute)